MODULE.bazel

```diff
@@ -21,7 +21,7 @@ use_repo(
 
 go_sdk = use_extension("//go:extensions.bzl", "go_sdk")
 go_sdk.download(name = "go_default_sdk", version = "1.18.3")
-use_repo(go_sdk, "go_default_sdk")
+use_repo(go_sdk, "go_default_sdk", "go_default_sdk_toolchains")
```

Comment:
I actually think we should be able to get rid of the `"go_default_sdk"` here. Could you give that a try?